docs/variants/protectli_ptx01/overview.md

```diff
@@ -22,8 +22,5 @@ TPM.
   open-source firmware update methods.
 * [Recovery](recovery.md) - gathers information on how to recover the platform
   from potential failure.
-* [Hardware configuration matrix](hardware-matrix.md) - describes the
-  platform's hardware configuration used during the Dasharo firmware
-  validation procedure.
-* [Test matrix](test-matrix.md) - describes validation scope used during
-  Dasharo firmware validation procedure.
```

Comment:
@Asiderr we need those two files in the current form. Please revert

Reply:
Fixed fbacd307